parser/parser.go

```diff
@@ -73,6 +74,21 @@ func Parse(path, fileNameParam string, params map[string]string) ([]byte, []inte
 	return data.Bytes(), errors, nil
 }
 
+//GetSchema request schema from the Parser service
```

Comment:
Você não precisa (necessariamente) dizer detalhes de implementação na documentação da API.

`// GetSchema returns the schema used to generate the parsed CSV.`

Reply:
Saquei